website/docs/r/schema_template_contract_filter.html.markdown

```diff
@@ -10,17 +10,21 @@ description: |-
 
 Manages MSO Schema Template Contract Filter.
 
+!> Do not use this resource together with resource [mso_schema_template_contract](https://registry.terraform.io/providers/CiscoDevNet/mso/latest/docs/resources/schema_template_contract).
```

Comment:
You cannot use it at all or just cannot use it if you define filters in the mso_schema_template_contract module?

Reply:
You cannot use it at all because a filter must be defined on the contract resource, else validation rule will get triggered. 

Error: "contract: contract1 in Schema: abr_filter , Template: abr_template_2 must be associated with filter(s)"{}